### ISSUE TITLE
Labels for objectness priors

### ISSUE THREAD

**Issue by fcdl94** [abearman/whats-the-point1#2]
Hi, sorry for opening an issue after a long time.

I was trying to implement your code in PyTorch with a different architecture, but I am stuck by finding a way of implementing the objectness prior. In particular, I don't know how to get the predictions which are used as ground truth.
Are these provided for PASCAL-VOC Images? How can I get them? 
I see that there is the Matlab code for them, but it's not clear to me on which data you had trained that model.

If you can provide the objectness prior labels as png/jpg images it would be amazing!
Thank you!

**Comment by fcdl94** [comment]
Ok, after looking carefully at the matlab code, I found that the model to get the objectness labels is already provided and one can make the labels just by running the scripts.
